node/src/command.rs

```diff
@@ -249,54 +247,52 @@ pub fn run() -> Result<()> {
             Ok(())
         }
```

Comment:
do we need to use benchmark? i.e. https://github.com/substrate-developer-hub/substrate-parachain-template/blob/master/node/src/command.rs#L236

Reply:
to be done in https://github.com/DataHighway-DHX/node/issues/228